docs/src/main/sphinx/connector/hive.rst

```diff
@@ -57,16 +58,28 @@ to using port 9083.
 General configuration
 ---------------------
 
-Create ``etc/catalog/example.properties`` with the following contents
-to mount the ``hive`` connector as the ``example`` catalog,
-replacing ``example.net:9083`` with the correct host and port
-for your Hive metastore Thrift service:
+To configure the Hive connector, create a catalog properties file
+``etc/catalog/myexample.properties`` that references the ``hive``
```

Comment:
```suggestion
``etc/catalog/example.properties`` that references the ``hive``
```

Reply:
@mosabua I'm changing all of these as requested, but I think it is a bad idea. It's confusing because the thrift URL is example.net, and it looks by name that they are related, no better than naming the catalog hive.properties, confusing it with the ``hive`` connector name... It would be a good project to change all of these catalog name examples to something like mycatalog.properties and all URLS to example.net so that nothing gets confused.